sprints/dashboard/models.py

```diff
@@ -124,12 +124,36 @@ def assignee_time(self) -> int:
 
         return max(self.time_estimate - self.review_time, 0)  # We don't want negative values here.
 
+    def get_review_time_from_story_points(self):
+        """
+        Get reviewer time for the task according to the SPRINT_HOURS_RESERVED_FOR_REVIEW setting and the story points
+        assigned
+        """
+        story_points_hours_map = settings.SPRINT_HOURS_RESERVED_FOR_REVIEW
+        if self.story_points == 0:
+            return story_points_hours_map.get("0")
+
+        if self.story_points < 2:
+            return story_points_hours_map.get("<2")
+
+        if self.story_points == 2:
+            return story_points_hours_map.get("2")
+
+        if self.story_points == 3:
+            return story_points_hours_map.get("3")
+
+        if self.story_points == 5:
+            return story_points_hours_map.get("5")
+
+        return story_points_hours_map.get(">5")
```

Comment:
It's not a flexible approach. Instead of hardcoding these values, we could define that undefined story points will be applied to the nearest defined ones. E.g.:
```python
{
    None: 2
    1.9: 0.5,
    2: 1,
    3: 2,
    5: 3,
    5.1: 5,
}
```
In this case, `0SP == 0.5h, 1SP == 0.5h, 1.99SP == 1h, 999SP == 5h, None == 2h`.

What do you think?

Reply:
Yep, I had my questions on this, now i realize that I misunderstood the example approach :) haha. ok, I'll work on that

Reply:
Changed the logic, so now goes to the closest/nearest defined  one